composer.json

```diff
@@ -1,17 +1,17 @@
-{
-    "name": "cardknox/cardknox",
-    "description": "Cardknox integration for Magento 2",
-    "type": "magento2-module",
-    "version": "1.0.13",
-    "license": [
-        "GPL-3.0"
-    ],
-    "autoload": {
-        "files": [
-            "registration.php"
-        ],
-        "psr-4": {
-            "CardknoxDevelopment\\Cardknox\\": ""
-        }
-    }
-}
+{

```

Comment:
What changed in this file?

Reply:
Nothing to changed in this file. I pushed code at that time this file updated by mistake.